lib/cisco_node_utils/node.rb

```diff
@@ -432,4 +432,61 @@ def find_subconfig(body, regex_query)
     subconfig.join("\n")
   end
   module_function :find_subconfig
+
```

Comment:
Shared helper methods like these belong in `cisco_cmd_utils.rb`, not `node.rb`.

https://github.com/cisco/cisco-network-node-utils/blob/develop/lib/cisco_node_utils/cisco_cmn_utils.rb